sde_collections/models/candidate_url.py

```diff
@@ -35,6 +35,7 @@ class CandidateURL(models.Model):
         blank=True,
         help_text="This is the original title scraped by Sinequa",
     )
+    scraped_text = models.TextField(blank=True, null=True)
```

Comment:
In the past, we have always done `default=""` for text fields.

Reply:
Convention followed.